src/ContainerCommitModal.jsx

```diff
@@ -74,23 +75,24 @@ class ContainerCommitModal extends React.Component {
         }
 
         const commitData = {};
-        commitData.name = this.props.container.id;
-        commitData.image_name = this.state.tag ? this.state.imageName + ":" + this.state.tag : this.state.imageName;
+        commitData.container = this.props.container.Id;
+        commitData.repo = this.state.imageName;
         commitData.author = this.state.author;
-        commitData.message = this.state.message;
         commitData.pause = this.state.pause;
         commitData.format = this.state.format;
 
+        if (this.state.comment)
+            commitData.comment = this.state.comment;
+
+        if (this.state.tag)
+            commitData.tag = this.state.tag;
+
         commitData.changes = [];
         if (this.state.command.trim() !== "") {
             let cmdData = "";
-            if (utils.compare_versions(this.props.version, "1.7.0") > -1) {
```

Comment:
That reminds me -- this kind of change that drops support for < 1.7 could land separately, as all our supported OSes have 1.9 now.

Reply:
oh right, it should, but thb. I am a bit unwilling now as I would cause conflicts for myself unnecessary.
Would you mind if I would keep it bundled in?

Reply:
No, it's fine, just in case this PR gets stalled for a bit longer.